### DISCUSSION TITLE
Hotfix not applying (new than 20231204) via SOUP

### DISCUSSION THREAD

**Discussion by ross-tsc** [Security-Onion-Solutions/securityonion#12114] [2.4] [answered]
### Version

2.4.30

### Installation Method

Security Onion ISO image

### Description

upgrading

### Installation Type

Distributed

### Location

cloud

### Hardware Specs

Exceeds minimum requirements

### CPU

8

### RAM

32

### Storage for /

256

### Storage for /nsm

SO AMI does not set up separate LV

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Hi there

We have a running distributed so 2.4.30 install running in AWS. We noticed two separate hotfixes came out since  20231204, but soup cannot see them. I suspect this is due to HOTFIX not being updated in the release. 

Is this intentional? If not, would you like me to throw you a PR?

Many thanks

R

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by TOoSmOotH** [comment, marked as answer]
The hotfix had to do with kickstart scripts on the ISO so there is nothing to apply on currently deployed grids.

**Reply by ross-tsc** [reply]
Excellent thank you
